vertical-pod-autoscaler/pkg/recommender/logic/recommender.go

```diff
@@ -26,6 +26,7 @@ var (
 	safetyMarginFraction = flag.Float64("recommendation-margin-fraction", 0.15, `Fraction of usage added as the safety margin to the recommended request`)
 	podMinCPUMillicores  = flag.Float64("pod-recommendation-min-cpu-millicores", 25, `Minimum CPU recommendation for a pod`)
 	podMinMemoryMb       = flag.Float64("pod-recommendation-min-memory-mb", 250, `Minimum memory recommendation for a pod`)
+	targetCPUPercentile  = flag.Float64("target-cpu-percentile", 0.9, "Target CPU percentile to consider when computing recommendations")
```

Comment:
I think this flag could use a more detailed description. I'll write my idea later.

Reply:
What do you think about the following description for the flag:

CPU usage percentile that will be used as a base for CPU target recommendation. Doesn't affect CPU lower bound, CPU upper bound nor memory recommendations.

?